### PR TITLE
Feature/empty state

### PR DESCRIPTION
**Pull request by HryshkoAnton**
add logic to show noNews component when there are no news

https://github.com/ita-social-projects/GreenCity/issues/1287

### CODE REVIEW

File: src/app/component/eco-news/components/no-news/no-news.component.html [hunk 1]
```diff
@@ -0,0 +1,17 @@
+<div class="container">
+    <div class="picture">
+        <img src="assets/img/noNews.jpg" alt="noNews">
```
Comment (RomanDeveloperAcc) [inline src/app/component/eco-news/components/no-news/no-news.component.html:3]: Please, put attributes in separate rows

File: src/app/component/eco-news/components/no-news/no-news.component.html [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+    </div>
+    <div class="description">
+        <div class="description__title">
+            <h2>
+                Unfotunately there is no news finded
```
Comment (RomanDeveloperAcc) [inline src/app/component/eco-news/components/no-news/no-news.component.html:8]: Please, think about the localization

File: src/app/component/eco-news/components/no-news/no-news.component.html [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+            </h2>
+        </div>
+        <div class="description__advise">
+            <p>
+                We couldn't find any news. Please change filters to get a result.
```
Comment (RomanDeveloperAcc) [inline src/app/component/eco-news/components/no-news/no-news.component.html:13]: Same as row №8

File: src/app/component/eco-news/components/no-news/no-news.component.html (hunk 1 above)
Comment (HryshkoAnton) [reply src/app/component/eco-news/components/no-news/no-news.component.html:3]: fixed

File: src/app/component/eco-news/components/no-news/no-news.component.html (hunk 3 above)
Comment (HryshkoAnton) [reply src/app/component/eco-news/components/no-news/no-news.component.html]: fixed

File: src/app/component/eco-news/components/no-news/no-news.component.html (hunk 2 above)
Comment (HryshkoAnton) [reply src/app/component/eco-news/components/no-news/no-news.component.html]: fixed

**Comment by mplesha** [comment]
Build Failed

**Review by RomanDeveloperAcc — APPROVED** [review]

**Review by nvolodymyr — APPROVED** [review]

File: src/app/component/eco-news/components/news-list/news-list.component.html [hunk 4]
```diff
@@ -21,7 +21,7 @@ <h1 class="main-header">{{ "homepage.eco-news.title" | translate }}</h1>
     <app-remaining-count [remainingCount]="remaining"></app-remaining-count>
     <app-change-view-button (view)="changeView($event)"></app-change-view-button>
   </div>
-  <div>
+  <div *ngIf="remaining">
```
Comment (mplesha) [inline src/app/component/eco-news/components/news-list/news-list.component.html:24]: Why go the extra `div`?

**Review by sofiiamatsyhin — APPROVED** [review]

**Pull request merged**